web/src/test/kotlin/no/nav/su/se/bakover/web/routes/grunnlag/OppholdIUtlandetVilkårJsonTest.kt

```diff
@@ -0,0 +1,70 @@
+package no.nav.su.se.bakover.web.routes.grunnlag
+
+import arrow.core.nonEmptyListOf
+import no.nav.su.se.bakover.common.serialize
+import no.nav.su.se.bakover.domain.grunnlag.OppholdIUtlandetGrunnlag
+import no.nav.su.se.bakover.domain.vilkår.OppholdIUtlandetVilkår
+import no.nav.su.se.bakover.domain.vilkår.Resultat
+import no.nav.su.se.bakover.domain.vilkår.VurderingsperiodeOppholdIUtlandet
+import no.nav.su.se.bakover.test.fixedTidspunkt
+import no.nav.su.se.bakover.test.getOrFail
+import no.nav.su.se.bakover.test.periode2021
+import org.junit.jupiter.api.Test
+import org.skyscreamer.jsonassert.JSONAssert
+import java.util.UUID
+
+internal class OppholdIUtlandetVilkårJsonTest {
+
+    @Test
+    fun `serialiserer vurdert opphold i utlandet`() {
+        JSONAssert.assertEquals(expectedOppholdIUtlandetVurdert, serialize(oppholdIUtlandet.toJson()), true)
+    }
+
+    @Test
+    fun `serialiserer ikke vurdert opphold i utlandet`() {
```

Comment:
Dette virker ikke å stemme helt overens med json som genereres for tilsvarende case i grunnlagsdata og vilkårsvurderinger? Ser ut som samme case blir `null` der - kan evt vurdere å endre denne til å svare med null?